apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -382,23 +338,6 @@ public final class AppConfiguration extends BaseAppConfiguration implements Seri
   public static Set<String> MICROMETER_CW_ALLOWED_METRIC_NAMES =
       Set.of("FissClaimRdaSink.change.latency.millis", "McsClaimRdaSink.change.latency.millis");
 
-  /**
-   * Instance of {@link MicrometerConfigHelper} used to create a {@link CloudWatchConfig} instance.
-   * Contains the property name to environment variable name mappings for supported {@link
-   * CloudWatchConfig} properties as well as default values for some environment variables.
-   */
-  @VisibleForTesting
-  static final MicrometerConfigHelper MICROMETER_CW_CONFIG_HELPER =
-      new MicrometerConfigHelper(
-          List.of(
-              new MicrometerConfigHelper.PropertyMapping(
-                  "cloudwatch.enabled", ENV_VAR_MICROMETER_CW_ENABLED, Optional.of("false")),
-              new MicrometerConfigHelper.PropertyMapping(
-                  "cloudwatch.namespace", ENV_VAR_MICROMETER_CW_NAMESPACE, Optional.empty()),
-              new MicrometerConfigHelper.PropertyMapping(
-                  "cloudwatch.step", ENV_VAR_MICROMETER_CW_INTERVAL, Optional.of("PT1M"))),
-          System::getenv);
-
```

Comment:
This is replaced by a static method later in this file so that a `ConfigLoader` can be used.